cli.js

```diff
@@ -24,7 +24,7 @@ const parser = yargs()
         glob: {
           desc: "Glob for finding plugins",
           type: "array",
-          default: ["plugins/*.(js|jsx|ts|tsx)"],
+          default: ["**/*.(js|jsx|ts|tsx)"],
```

Comment:
```suggestion
          default: ["(plugins|content)/**/*.(js|jsx|ts|tsx)"],
```
I'm thinking about something like this, but I want to figure out some other plugin repo cleanup first.